Tests/UnitTests/Networking/Backend/__Snapshots__/BackendGetIntroEligibilityTests/watchOS-testPostsProductIdentifiers.1.json

```diff
@@ -1,7 +1,6 @@
 {
   "headers" : {
     "Authorization" : "Bearer asharedsecret",
-    "content-type" : "application/json",
```

Comment:
I'm guessing we don't really need the updates to the files that have the same content, right? only the addition of the new ones?

Reply:
We could remove them for the sake of making the review easier. Even though the contents are functionally the same, since the order they appear in is different, the tests fail without these changes present.